Add comment about documentation for config.xml.
Also add mappings for matroska. Also add mappings for ogg in like manner as
audio files.
==========================================================================
--- a/src/config_manager.cc
+++ b/src/config_manager.cc
@@ -426,6 +426,13 @@
     config->setAttribute(_("xmlns"), _(XML_XMLNS) + CONFIG_XML_VERSION);
     config->setAttribute(_("xmlns:xsi"), _(XML_XMLNS_XSI));
     config->setAttribute(_("xsi:schemaLocation"), _(XML_XMLNS) + CONFIG_XML_VERSION + " " + XML_XMLNS + CONFIG_XML_VERSION + ".xsd");
+
+    Ref<Comment> docinfo(new Comment(_("\n\
+     Read /usr/share/doc/mediatomb-common/README.gz section 6 for more\n\
+     information on creating and using config.xml configration files.\n\
+    "), true));
+    config->appendChild(RefCast(docinfo, Node));
+
     Ref<Element> server(new Element(_("server")));
    
     Ref<Element> ui(new Element(_("ui")));
@@ -614,6 +621,8 @@
                                _(UPNP_DEFAULT_CLASS_VIDEO_ITEM)));
     mtupnp->appendElementChild(map_from_to(_("image/*"), 
                                _("object.item.imageItem")));
+    mtupnp->appendElementChild(map_from_to(_("application/ogg"),
+                               _(UPNP_DEFAULT_CLASS_MUSIC_TRACK)));
 
     mappings->appendElementChild(mtupnp);
 
@@ -644,6 +653,10 @@
                                            _(CONTENT_TYPE_DVD)));
     mtcontent->appendElementChild(treat_as(_("application/x-iso9660-image"), 
                                            _(CONTENT_TYPE_DVD)));
+    mtcontent->appendElementChild(treat_as(_("video/x-matroska"),
+                                           _(CONTENT_TYPE_MKV)));
+    mtcontent->appendElementChild(treat_as(_("audio/x-matroska"),
+                                           _(CONTENT_TYPE_MKA)));
 
     mappings->appendElementChild(mtcontent);
     import->appendElementChild(mappings);
--- a/src/metadata_handler.h
+++ b/src/metadata_handler.h
@@ -57,6 +57,8 @@
 #define CONTENT_TYPE_MPEG       "mpeg"
 #define CONTENT_TYPE_DVD        "dvd"
 #define CONTENT_TYPE_QUICKTIME  "quicktime"
+#define CONTENT_TYPE_MKV        "mkv"
+#define CONTENT_TYPE_MKA        "mka"
 
 #define OGG_THEORA              "t"
 
